### DISCUSSION TITLE
Is Index Removed

### DISCUSSION THREAD

**Discussion by ankurparashar** [MichaelCade/90DaysOfDevOps#254] [Ideas]
Recently when I started 90DaysOfDevOps there was an index page which tells about each day title, I believe it is missing since 2022 and 2023 folders has made, I believe indexing should be maintained, since it is important as
-  Someone is doing self-paced learning
-  Need to read about specific topic in detail
-  Wants to contribute to a topic but unable to find the direct path

Do let me know your thoughts since it is very topic as it is increasing drop rates.

**Comment by MichaelCade** [comment]
From the main repository we still have the readme.md and you will see in there hopefully 2 images with 2022 and 2023 these lead to 2022.md and 2023.md 

The idea behind this was to enable a better user experience when people are following all content. 

I know this was a late reply as well.
